scripts/db.js

```diff
@@ -110,22 +110,22 @@ DB.prototype.changes = function (dbName, params) {
 
 DB.prototype.changesArray = function (dbName, params) {
   return this._slouch._req({
-    url: this._slouch._url + '/' + dbName + '/_changes',
+    url: this._slouch._url + '/' + encodeURIComponent(dbName) + '/_changes',
     qs: params,
     parseBody: true
   });
 };
 
 DB.prototype.view = function (dbName, viewDocId, view, params) {
   return new CouchPersistentStreamIterator({
-    url: this._slouch._url + '/' + dbName + '/' + viewDocId + '/_view/' + view,
+    url: this._slouch._url + '/' + encodeURIComponent(dbName) + '/' + viewDocId + '/_view/' + view,
     qs: params
   }, 'rows.*');
 };
 
 DB.prototype.viewArray = function (dbName, viewDocId, view, params) {
   return this._slouch._req({
-    url: this._slouch._url + '/' + dbName + '/' + viewDocId + '/_view/' + view,
+    url: this._slouch._url + '/' + encodeURIComponent(dbName) + '/' + viewDocId + '/_view/' + view,
```

Comment:
Does `viewDocId` need to be encoded as well?

Reply:
Probably yes. I'll make the changes.